libs/api/chat/repository/feature/src/lib/chat.repository.ts

```diff
@@ -1,26 +1,87 @@
 import { Injectable } from '@nestjs/common';
 import { PrismaService } from '@charity-spot/api/shared/services/prisma';
-import { ClientLogin } from '@charity-spot/client/login';
 
 @Injectable()
 export class ChatRepository {
   constructor(private prisma: PrismaService) {}
 
-  async AddMessages(orgID, clientID : string ,text: string)
+  //create an empty thread for messages
+  async createThread(orgID, clientID : string)
   {
     const u = await this.prisma.chatHistory.create({
       data:
       {
         OrgID: orgID,
-        ClientID:clientID,
-        Messages:text
+        ClientID:clientID
       }
     })
 
     return u;
   }
```

Comment:
This will then just send a message without any hustle and I will use other functions to update other fields independently